worker-test/pom.xml

```diff
@@ -88,13 +88,6 @@
 
     <build>
         <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <skip>true</skip>
-                </configuration>
-            </plugin>
```

Comment:
This was added to stop ShutdownDeveloperTest from running.  Since its been renamed ShutdownDeveloperIT its no longer needed.